app/assets/javascripts/my_modules/repositories.js

```diff
@@ -756,6 +756,14 @@ var MyModuleRepositories = (function() {
       UPDATE_REPOSITORY_MODAL.data('update-url', data.update_url);
       UPDATE_REPOSITORY_MODAL.modal('show');
     });
+
+    setTimeout(() => {
+      $('.hide-assign-repo-modal').on('click', function() {
+        setTimeout(() => {
+          $('body').addClass('modal-open');
+        }, 0);
+      });
+    }, 200)
```

Comment:
Could this maybe be bound to a `shown.bs.modal` event or something similar, to avoid timeouts?

Reply:
> Could this maybe be bound to a `shown.bs.modal` event or something similar, to avoid timeouts?

I tried it but didnt seem to work, timeouts were the only solution i could find to be honest